2021-10-08 13:04:45,784 INFO    MainThread:85397 [wandb_setup.py:_flush():69] setting env: {'entity': 'rishanthrajendhran', 'project': 'eigengame', 'sweep_id': 'ggcsznof', 'root_dir': '/Users/rishanthrajendhran/Desktop/EigenGame', 'run_id': '0lrrkmvs', 'sweep_param_path': '/Users/rishanthrajendhran/Desktop/EigenGame/wandb/sweep-ggcsznof/config-0lrrkmvs.yaml'}
2021-10-08 13:04:45,785 INFO    MainThread:85397 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-08 13:04:45,785 INFO    MainThread:85397 [wandb_init.py:_log_setup():357] Logging user logs to /Users/rishanthrajendhran/Desktop/EigenGame/wandb/run-20211008_130445-0lrrkmvs/logs/debug.log
2021-10-08 13:04:45,785 INFO    MainThread:85397 [wandb_init.py:_log_setup():358] Logging internal logs to /Users/rishanthrajendhran/Desktop/EigenGame/wandb/run-20211008_130445-0lrrkmvs/logs/debug-internal.log
2021-10-08 13:04:45,785 INFO    MainThread:85397 [wandb_init.py:init():390] calling init triggers
2021-10-08 13:04:45,785 INFO    MainThread:85397 [wandb_init.py:init():397] wandb.init called with sweep_config: {'learningRate': 0.01, 'numStepsPerIteration': 50}
config: {'xDim': (10000, 10), 'numIterations': 35000, 'numStepsPerIteration': 100, 'k': 10, 'learningRate': 1, 'flags': ['-adagrad'], 'isSymmetric': False, 'variant': 'c', 'tolerance': 10, 'distanceTolerance': 0.01}
2021-10-08 13:04:45,785 INFO    MainThread:85397 [wandb_init.py:init():435] starting backend
2021-10-08 13:04:45,785 INFO    MainThread:85397 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-08 13:04:45,804 INFO    MainThread:85397 [backend.py:ensure_launched():135] starting backend process...
2021-10-08 13:04:45,814 INFO    MainThread:85397 [backend.py:ensure_launched():140] started backend process with pid: 85411
2021-10-08 13:04:45,818 INFO    MainThread:85397 [wandb_init.py:init():440] backend started and connected
2021-10-08 13:04:45,820 INFO    MainThread:85397 [wandb_run.py:_config_callback():878] config_cb None None {'learningRate': 0.01, 'numStepsPerIteration': 50}
2021-10-08 13:04:45,821 INFO    MainThread:85397 [wandb_init.py:init():499] updated telemetry
2021-10-08 13:04:45,821 INFO    MainThread:85397 [wandb_init.py:init():522] communicating current version
2021-10-08 13:04:47,305 INFO    MainThread:85397 [wandb_init.py:init():527] got version response 
2021-10-08 13:04:47,305 INFO    MainThread:85397 [wandb_init.py:init():535] communicating run to backend with 30 second timeout
2021-10-08 13:04:47,665 INFO    MainThread:85397 [wandb_init.py:init():562] starting run threads in backend
2021-10-08 13:04:52,712 INFO    MainThread:85397 [wandb_run.py:_console_start():1651] atexit reg
2021-10-08 13:04:52,716 INFO    MainThread:85397 [wandb_run.py:_redirect():1525] redirect: SettingsConsole.REDIRECT
2021-10-08 13:04:52,727 INFO    MainThread:85397 [wandb_run.py:_redirect():1530] Redirecting console.
2021-10-08 13:04:52,733 INFO    MainThread:85397 [wandb_run.py:_redirect():1586] Redirects installed.
2021-10-08 13:04:52,733 INFO    MainThread:85397 [wandb_init.py:init():587] run started, returning control to user process
2021-10-08 13:11:49,428 INFO    MainThread:85397 [wandb_run.py:finish():1311] finishing run rishanthrajendhran/eigengame/0lrrkmvs
2021-10-08 13:11:49,430 INFO    MainThread:85397 [wandb_run.py:_atexit_cleanup():1621] got exitcode: 0
2021-10-08 13:11:49,432 INFO    MainThread:85397 [wandb_run.py:_restore():1593] restore
2021-10-08 13:11:52,209 INFO    MainThread:85397 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 825
}

2021-10-08 13:11:52,629 INFO    MainThread:85397 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 825
}

2021-10-08 13:11:53,191 INFO    MainThread:85397 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 8114
}

2021-10-08 13:11:53,303 INFO    MainThread:85397 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 17491
}

2021-10-08 13:11:53,416 INFO    MainThread:85397 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 17491
}

2021-10-08 13:11:53,526 INFO    MainThread:85397 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 17491
}

2021-10-08 13:11:53,635 INFO    MainThread:85397 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 17491
}

2021-10-08 13:11:53,743 INFO    MainThread:85397 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17491
  total_bytes: 17491
}

2021-10-08 13:11:53,852 INFO    MainThread:85397 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17491
  total_bytes: 17491
}

2021-10-08 13:11:53,962 INFO    MainThread:85397 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17491
  total_bytes: 17491
}

2021-10-08 13:11:54,069 INFO    MainThread:85397 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17491
  total_bytes: 17491
}

2021-10-08 13:11:54,180 INFO    MainThread:85397 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17491
  total_bytes: 17491
}

2021-10-08 13:11:54,291 INFO    MainThread:85397 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17491
  total_bytes: 17491
}

2021-10-08 13:11:54,402 INFO    MainThread:85397 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17491
  total_bytes: 17491
}

2021-10-08 13:11:55,032 INFO    MainThread:85397 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17491
  total_bytes: 17491
}

2021-10-08 13:11:55,731 INFO    MainThread:85397 [wandb_run.py:_wait_for_finish():1746] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17491
  total_bytes: 17491
}
local_info {
}

2021-10-08 13:11:57,267 INFO    MainThread:85397 [wandb_run.py:_append_history():1959] rendering history
2021-10-08 13:11:57,268 INFO    MainThread:85397 [wandb_run.py:_append_summary():1914] rendering summary
2021-10-08 13:11:57,268 INFO    MainThread:85397 [wandb_run.py:_append_files():2009] logging synced files
